### DISCUSSION TITLE
Lesson 1 challenge : How to solve?

### DISCUSSION THREAD

**Discussion by nazzal5448** [Cyfrin/foundry-full-course-cu#1565] [Q&A]
I have watched the lessons, but I don't know how to even start the challenge. please guide me step by step as how to go about it. its my first time so I will really appreciate proper guidance.

**Comment by ARomano79** [comment]
Hi Nazzal, which module are you studying?

**Comment by EngrPips** [comment]
Hello @nazzal5448 ,  what  lesson challenge are you attempting to solve ?

**Comment by kartik01112004** [comment]
hello
i wanted to do challenge which were provided at the and of basics of blockchain but i dont understand what to do and how to do 
i visited the website provided by Qr code 
it have some contracts but what to do after that?

**Reply by EngrPips** [reply]
Call the `solveChallenge` method on the contract with the required arguments and if you do that properly it should mint an NFT to your address.

**Reply by kartik01112004** [reply]
i called the function with required args transaction was successful but i didnt get any nft ..
i called fn from write contract button

**Reply by EngrPips** [reply]
The NFT will not automatically appear on your Metamask wallet; you have it already, but you need to manually add it by clicking on the `import NFT` option on your Metamask wallet. Then, you will provide the necessary details to get the NFT to show in your wallet


<img width="357" alt="Screenshot 2024-06-02 at 07 10 57" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/95760918/f196c615-4516-4251-b5d1-6ab9e6e5b26f">

**Reply by kartik01112004** [reply]
ohh ok ok

**Comment by mlntdtechbae** [comment]
<img width="295" alt="Screenshot 2024-07-08 at 9 10 16 AM" src="https://github.com/Cyfrin/foundry-full-course-cu/assets/112206621/9bbca193-7615-4fbc-af64-ed5782adfe73">

Can everyone else see their NFT image? It's blank for me.

**Reply by kartik01112004** [reply]
check this discussion this worked for me 
[link](https://github.com/Cyfrin/foundry-full-course-cu/discussions/364#discussion-5411092)

**Reply by EngrPips** [reply]
boom!
